content/api/hub/sample_hugo.md

```diff
@@ -12,7 +12,7 @@ language = "Rust"
 created_at = "2023-08-31T10:52:54Z"
 last_updated = "2023-08-31T00:52:54Z"
 spin_version = ">v1.3"
-summary = "A template repository you can use for serving your Hugo site with Fermyon Spin and for deploying it to Fermyon Cloud"
+summary = "A template for deploying your Hugo site using Fermyon Spin and Cloud."
```

Comment:
this is supremely pedantic _even by my standards_ but most hub descriptions seem _not_ to end with a full stop (I guess because sentence fragment rather than full sentence)